config/locales/errors/en.yml

```diff
@@ -9,12 +9,13 @@ en:
     messages:
       already_confirmed: was already confirmed, please try signing in
       blank: Please fill in this field.
+      confirmation_code_incorrect: Incorrect code. Did you type it in correctly?
       confirmation_invalid_token: >
-        Invalid confirmation link. Either the link expired or you 
-        already confirmed your account. 
+        Invalid confirmation link. Either the link expired or you
+        already confirmed your account.
       confirmation_period_expired: >
         Expired confirmation link.
-        You can click "Resend confirmation instructions" to get another one. 
+        You can click "Resend confirmation instructions" to get another one.
```

Comment:
My bad. I was looking at the wrong page. I just realized this change is not related to this PR. It looks like it's just cleaning up whitespace. It turns out that to use this magic trick, the only thing you can put as the value is the key you are referencing, like this:
```yaml
confirmation_period_expired: :idv.messages.usps.send_again
```
So, it doesn't look like we can use it here 😢